admin/nyx/Makefile

```diff
@@ -0,0 +1,37 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nyx
+PKG_VERSION:=2.1.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=nyx
+PKG_HASH:=88521488d1c9052e457b9e66498a4acfaaa3adf3adc5a199892632f129a5390b
+
+PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
+PKG_LICENSE:=GPL-3.0
```

Comment:
This is deprecated Identifier, please use new one from https://spdx.org/licenses/

Reply:
> This is deprecated Identifier, please use new one from https://spdx.org/licenses/

Updated